curvefs/src/client/metaserver_client.cpp

```diff
@@ -17,65 +17,301 @@
 
 /*
  * Project: curve
- * Created Date: Thur May 27 2021
- * Author: xuchaojie
+ * Created Date: Thur Jun 15 2021
+ * Author: lixiaocui
  */
 
 #include "curvefs/src/client/metaserver_client.h"
 
 namespace curvefs {
 namespace client {
 
-CURVEFS_ERROR MetaServerClientImpl::GetDentry(uint32_t fsId, uint64_t inodeid,
-              const std::string &name, Dentry *out) {
-    return CURVEFS_ERROR::OK;
+CURVEFS_ERROR
+MetaServerClientImpl::MetaServerRPCExcutor::DoRPCTask(RPCFunc task) {
+    brpc::Channel channel;
+    int ret = channel.Init(opt_.msaddr.c_str(), nullptr);
+    if (ret != 0) {
+        LOG(WARNING) << "Init channel failed, addr = " << opt_.msaddr;
+        return static_cast<CURVEFS_ERROR>(-EHOSTDOWN);
```

Comment:
handle this errcode here，do not use static_cast

Reply:
yes, mark todo with exception handling